src/gluonts/torch/model/deepar/estimator.py

```diff
@@ -95,6 +95,10 @@ class DeepAREstimator(PyTorchLightningEstimator):
         Number of RNN layers (default: 2).
     hidden_size
         Number of RNN cells for each layer (default: 40).
+    lr
```

Comment:
Can we use `learning_rate` to be consistent with our `mx` version?

Reply:
`mx` version doesn't use `learning_rate` directly in estimator, but rather in `mx` trainer. So as @lostella mentioned, I think `lr` is consistent with `torch` trainer/optimizer. And to make `mx` and `torch` interfaces consistent, we can use `lr` in `mx` trainer as well.